### DISCUSSION TITLE
Welcome to vscode-fortran-support Discussions!

### DISCUSSION THREAD

**Discussion by gnikit** [krvajal/vscode-fortran-support#427] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SomeGuy5391** [comment]
Hello. I am Ken Levine, an old guy. I last programmed in Fortran about 30 years ago and having issues with vs code. How do I tell if I have gFortran on my machine?
